.github/workflows/ci.yml

```
@@ -73,6 +73,29 @@ jobs:
           cd tmp
           ruby test/run.rb
 
+  # Don't run tests on TruffleRuby and JRuby because tests don't pass on them yet.
+  # So check only gem installing.
+  install:
+    name: >-
+      Install ${{ matrix.ruby }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - truffleruby
+          - jruby
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
+
+      - run: rake install
```

```suggestion
      - run: rake install
      - run: ruby -e 'require "fiddle"'
```
like @kou suggested in https://github.com/ruby/fiddle/pull/146#discussion_r1765898874 seems good to at least check it requires successfully.